client/my-sites/comments/style.scss

```diff
@@ -90,7 +104,7 @@
 
 	&.comment-list__transition-leave-active {
 		opacity: 0.01;
-		transition: opacity .15s linear;
+		transition: opacity 0.15s linear;
```

Comment:
Oh does the leading 0 make a difference or was it the formatter?

Reply:
Definitely the formatter.